base_changeset/security/rules.xml

```diff
@@ -16,4 +16,16 @@
             name="domain_force"
         >['|',('company_id','=',False),('company_id', 'in', user.company_ids.ids)]</field>
     </record>
+    <record id="rule_record_changeset_change_base_user" model="ir.rule">
+        <field name="name">Restrict changeset change access for users</field>
+        <field name="model_id" ref="model_record_changeset_change" />
+        <field name="groups" eval="[(4, ref('base.group_user'))]" />
+        <field name="domain_force">[('create_uid', '=', user.id)]</field>
+    </record>
+    <record id="rule_record_changeset_change_user" model="ir.rule">
+        <field name="name">Allow changeset change access for changeset users</field>
+        <field name="model_id" ref="model_record_changeset_change" />
+        <field name="groups" eval="[(4, ref('group_changeset_user'))]" />
+        <field name="domain_force">[('1', '=', '1')]</field>
```

Comment:
> you can only see your own changes if you're not one of the validating users

In case you want to propose this change in OCA, the idea is reasonable but it would probably be not fitting for all of our customers. I guess there will be a debate in OCA. But I would prefer to have an opinion from my colleagues who are actually having relationship with our customers.

Reply:
it is my plan to add this to the OCA (but not in the migration PR because I don't want to keep resetting the reviews there). I came up with this when searching for a way to not impact current users, so I'd indeed be curious how it's used in the wild now that this addition would interfere with.

But if you generally with this but there's reasons not to add it in base, I'll happily do so in `vanmoof_changeset`

Reply:
I won't update the current migration PR anyways, because I want this to be merged as-is and then add security and the backport of the list view display. Let's see what @StefanRijnhart thinks about the security approach

Reply:
I really like this approach of having per-rule access rules. You could consider adding an onchange on the model field to add the groups for which there is an ir.model.access for that model.

Reply:
I don't get the last sentence. If we have validator groups per rule, why would we want to fill it with all groups that have access to the model? Validators should be few in some specialized group I think

Reply:
You are right, I was confusing two different concepts.